Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -1801,6 +1837,56 @@ def apply_issue_transition(client: JiraBaseClient, issue_id_or_key: str, transit
     raise DemistoException(f'Transition "{transition_name}" not found. \nValid transitions are: {transitions_name} \n')
 
 
+def get_issue_forms(client: JiraBaseClient, issue_id: str) -> tuple[List, List]:
+    """Gets the forms from the client and processes them into a usable JSON format.
+
+    :param client: Client to make the API call with
+    :type client: JiraBaseClient
+    :param issue_id: Issue ID to get the forms for
+    :type issue_id: str
+    :return: The raw JSON response and the formatted outputs
+    :rtype: tuple[List, List]
+    """
+    response = client.issue_get_forms(issue_id=issue_id)
+    demisto.debug(f'Response from Jira API rest/proforma/api/2/issues/{issue_id}/forms: {json.dumps(response)}')
+    outputs = []
+    for form in response:
+        questions = []
+        for q_id, q_data in form.get('design', {}).get('questions').items():
```

Comment:
```suggestion
        for question_id, question_data in form.get('design', {}).get('questions').items():
```
meaningful naming please